README.md

```diff
@@ -1 +1,2 @@
-# java-mentoring
\ No newline at end of file
+# java-mentoring
+# improving
```

Comment:
Please, include a better description of this repository. Read https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax